src/test/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesTestUtil.java

```diff
@@ -181,7 +176,9 @@ public static boolean isWindows() {
      * Labels to add to the pods so we can match them to a specific build run, test class and method
      */
     public static Map<String, String> getLabels(KubernetesCloud cloud, Object o, TestName name) {
-        HashMap<String, String> l = Maps.newHashMap(DEFAULT_LABELS);
+        Map<String, String> l = new HashMap<>();
+        l.put(BRANCH_NAME, BRANCH_NAME == null ? "undefined" : BRANCH_NAME);
+        l.put(BUILD_NUMBER, BUILD_NUMBER == null ? "undefined" : BUILD_NUMBER);
```

Comment:
This should fix the jackson problem.
```suggestion
        l.put("BRANCH_NAME", BRANCH_NAME == null ? "undefined" : BRANCH_NAME);
        l.put("BUILD_NUMBER", BUILD_NUMBER == null ? "undefined" : BUILD_NUMBER);
```